### DISCUSSION TITLE
Clock accuracy

### DISCUSSION THREAD

**Discussion by ioukos** [pvvx/ATC_MiThermometer#508] [General]
On the 10th of April I set clock on LYWSD03MMC.
Today on the 21st : the clock is 6 minutes ahead of "real" (on smartphone) time.

Just to let you know. 
I think it's hardware issue.

**Comment by pvvx** [comment]
https://github.com/pvvx/ATC_MiThermometer/issues/56

**Reply by ioukos** [reply]
OK thanks, sorry for inconvenience.
